src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -156,6 +156,8 @@ set_defaults()
   this->only_2D = 0;
   this->kernelised_output_filename_prefix="";
   this->hybrid=0;
+  this->only_iterative_info=0;
+  this->_disable_output=1;
```

Comment:
Is this a bugfix or new feature? Should be highlighted separately in CHANGES.md?